src/server/src/main/java/io/cassandrareaper/resources/RepairScheduleResource.java

```diff
@@ -52,6 +51,7 @@
 
 import com.google.common.base.Preconditions;
 import com.google.common.collect.Lists;
+import io.dropwizard.jersey.PATCH;
```

Comment:
@adejanovski to make Checkstyle happy I had to go about changing a bunch of imports that IntelliJ had automatically organized, which I found a bit silly :-)

I've fixed things here in this PR, so we don't have to worry about it at the moment, but what would you think about removing this Checkstyle rule?

`<property name="customImportOrderRules" value="SAME_PACKAGE(2)###STANDARD_JAVA_PACKAGE###THIRD_PARTY_PACKAGE###STATIC"/>`

That enforces import ordering -- my feeling is that we should let the IDEs do their job and organize for us.  Or perhaps think about moving to something like https://github.com/google/google-java-format that can actively enforce formatting, would probably be nice to do similar in the UI code with prettier as well.  What do you think?

Reply:
I understand your frustration here but I'm reluctant to change the formatting rules to rely on IDEs instead. They will most probably apply different orderings which will create some noise in PRs as there's gonna be some unnecessary line changes.

Instead, we could maybe make it so that the rule matches IntelliJ's ordering of imports: 

```
To configure the check so that it matches default IntelliJ IDEA formatter configuration (tested on v14):

group of static imports is on the bottom
groups of non-static imports: all imports except of "javax" and "java", then "javax" and "java"
imports will be sorted in the groups
groups are separated by one blank line
Note: "separated" option is disabled because IDEA default has blank line between "java" and static imports, and no blank line between "javax" and "java"

<module name="CustomImportOrder">
  <property name="customImportOrderRules"
    value="THIRD_PARTY_PACKAGE###SPECIAL_IMPORTS###STANDARD_JAVA_PACKAGE###STATIC"/>
  <property name="specialImportsRegExp" value="^javax\."/>
  <property name="standardPackageRegExp" value="^java\."/>
  <property name="sortImportsInGroupAlphabetically" value="true"/>
  <property name="separateLineBetweenGroups" value="false"/>
</module>
```

this was pasted from the checkstyle docs: https://checkstyle.org/config_imports.html#ImportOrder_Examples

Apparently there's some workaround for the java/javax imports which can't be dealt with appropriately by IntelliJ: https://stackoverflow.com/a/64243861/5106246